composer/loggers/remote_uploader_downloader.py

```diff
@@ -18,24 +18,32 @@
 import warnings
 from multiprocessing.context import SpawnProcess
 from typing import Any, Callable, Dict, List, Optional, Set, Tuple, Type, Union
+from urllib.parse import urlparse
 
 from composer.core.state import State
 from composer.loggers.logger import Logger
 from composer.loggers.logger_destination import LoggerDestination
 from composer.utils import ObjectStore, ObjectStoreTransientError, dist, format_name_with_dist, get_file, retry
+from composer.utils.object_store import LibcloudObjectStore, S3ObjectStore, SFTPObjectStore
```

Comment:
(minor) nit: Does this need a tested import or can we add to line above and flatten imports?